### DISCUSSION TITLE
Deploy to PROD my new admin panel.

### DISCUSSION THREAD

**Discussion by gequz** [Laravel-Backpack/community-forum#1069] [General]
Hi,

I installed DevTools as dev dependency and created a new Cast model:

<img width="417" alt="Screenshot 2024-07-07 at 13 33 24" src="https://github.com/Laravel-Backpack/community-forum/assets/53155966/249ae57b-5aa7-4de9-83a8-b1f93198240b">

Now, I'd like to deploy my new model Cast and its CRUD forms to PROD:

<img width="914" alt="Screenshot 2024-07-07 at 13 33 40" src="https://github.com/Laravel-Backpack/community-forum/assets/53155966/ba7fd8fe-3250-4bfe-bf14-3e40602e44f0">

Is there a detailed step-by-step guide of how to do this? I'd like my PROD users not to see the existing models, but only the new Cast model and its crud forms.

many thanks

**Comment by jcastroa87** [comment]
Hi @gequz 

It is deploy like any Laravel app

1. Set your env params.
```
APP_ENV=production
APP_DEBUG=false
APP_URL=https://yourdomain.com
```
2. Run install command:
```
composer install --optimize-autoloader --no-dev
```
3. Set cache
```
php artisan optimize:clear
php artisan basset:clear
php artisan basset:cache
php artisan optimize
```

If you need to protect some Modules, you can use [Permission Manager](https://github.com/Laravel-Backpack/PermissionManager)

Cheers.

**Reply by gequz** [reply]
Thanks @jcastroa87 , I did follow your guidance.

My new model Casts appears, and I can see the list view.
However, when I try to Add new Cast I see a 500 error page and the following error appears in the log:

`[2024-07-08 21:42:19] local.ERROR: Cannot find the field view: ckeditor. Please check for typos. If you are trying to use a PRO field, please first purchase and install the backpack/pro addon from backpackforlaravel.com (View: vendor/backpack/crud/src/resources/views/crud/inc/show_fields.blade.php)`

I do have the PRO license on my local dev machine.
I never deployed backpack to server before.

Are we missing a command?

**Comment by gequz** [comment]
Thanks @jcastroa87 all sorted now.

I had to move backpack/pro from dev to require in `composer.json` and then it works like a charm.

Thank you for your steps above, they are very useful!
